### PR TITLE
Newmain

### PR DESCRIPTION
**Pull request by yuvalShalgi227**

### CODE REVIEW

File: Java/main-sample/src/main/java/com/perfectomobile/sample/SimpleTest.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+            reportiumClient.stepEnd();
+
+            //STOP TEST
+            testResult = TestResultFactory.createSuccess();
+
+            driver.close();
+        } catch (Exception e) {
+            e.printStackTrace();
+        } finally {
+            reportiumClient.testStop(testResult);
```
Comment (archmisha) [inline Java/main-sample/src/main/java/com/perfectomobile/sample/SimpleTest.java]: dont do any operations after driver close.

Comment (yuvalShalgi227) [reply Java/main-sample/src/main/java/com/perfectomobile/sample/SimpleTest.java]: ok, fixed, can we merge i don't get what the conflict is about..

**Comment by yuvalShalgi227** [comment]
Conflicting files won't allow me, don't see a reason for this...

**Comment by archmisha** [comment]
Because you are not sync with master

**Comment by yuvalShalgi227** [comment]
can't sync it, need help

**Comment by archmisha** [comment]
Well you need to use git extentions :)
Anyway i looked at your branch, its from 1 Apr. And since then Iddo changed
the MyApp sample to support the VCS integration feature.
You deleted that file and created a new one instead without all his
changes...

I think you need to take the current master and apply your changes there,
no point working on so old master version from 4 month ago...
Also dont delete the file, just modify it

Best regards,
Michael Dikman

**Comment by yuvalShalgi227** [comment]
ok, but does any one run iddo samples ? i deleted onlymyapp now

**Comment by archmisha** [comment]
myapp contains iddo vcs samples inside with custom fields

**Pull request merged**